### PR TITLE
Add Vienna.

### PR DESCRIPTION
**Pull request by codecop**

### CODE REVIEW

File: _data/events/virtual-2020-11-07-coderetreat-gdcr-2020-vienna-remote.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "date": {
+    "start": "2020-11-07T09:00:00+01:00",
+    "end": "2020-11-07T17:00:00+01:00"
+  },
+  "moderators": [
+    {
+      "name": "tbd"
+    }
+  ],
+  "sponsors": [
```
Comment (rradczewski) [inline _data/events/virtual-2020-11-07-coderetreat-gdcr-2020-vienna-remote.json:16]: Should be left empty (`[]`)

File: _data/events/virtual-2020-11-07-coderetreat-gdcr-2020-vienna-remote.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "title": "Coderetreat GDCR 2020 Vienna (remote)",
+  "format": "classic",
+  "spoken_language": "English",
+  "url": "https://www.softwerkskammer.org/activities/gdcr20-vienna",
+  "code_of_conduct": "https://makers.tech/code-of-conduct/",
+  "date": {
+    "start": "2020-11-07T09:00:00+01:00",
+    "end": "2020-11-07T17:00:00+01:00"
+  },
+  "moderators": [
```
Comment (rradczewski) [inline _data/events/virtual-2020-11-07-coderetreat-gdcr-2020-vienna-remote.json:11]: Should be left empty (`[]`)

**Review by rradczewski — CHANGES_REQUESTED** [review]

**Comment by codecop** [comment]
Done

**Comment by rradczewski** [comment]
Hi, we hope you had a great Global Day Of Coderetreat! We're trying to understand how the current situation has affected the events this year and how we could support you next year for #GDCR2021.

For this purpose, we created [a seven-questions-survey](https://docs.google.com/forms/d/e/1FAIpQLSdi8fmw42s8tXtVneJH5VcRpB9vMf35gJ2APw4PijQ0-Gq_Ww/viewform?usp=sf_link) (anonymously, all optional, single page) for those who organized an event this year. It would help us tremendously if you could fill it out. It doesn't take long, promise :smile: 

Thanks and all the best,
Raimo

**Pull request merged**